client/src/contexts/LoginContext.js

```diff
@@ -3,11 +3,11 @@ import jwt from "jsonwebtoken";
 export const LoginContext = createContext();
 
 export const LoginProvider = props => {
-  const token = localStorage.getItem("token");
+  const token = sessionStorage.getItem("token");
   const [login, setLogin] = useState(token === null ? "loggedOut" : "loggedIn");
   const [user, setUser] = useState(token === null ? null : jwt.decode(token));
   const Logout = history => {
-    localStorage.removeItem("token");
+    sessionStorage.removeItem("token");
```

Comment:
I'm not an expert on session vs localstorage but why session here?

Reply:
I didn't want the user to logged in when they close the browser or comeback. The next step would be to check the expiration of the token. Have to add this to the ticket.